source/conversions.tex

```diff
@@ -524,21 +524,21 @@
 object. The null pointer value is converted to the null pointer value of
 the destination type.
 
-\rSec1[conv.mem]{Pointer to member conversions}
+\rSec1[conv.mem]{Pointer-to-member conversions}
```

Comment:
I'm concerned that this change suggests a false parallel: previously, all the "X-to-Y conversion"s converted an X into a Y. A pointer to member conversion definitely does not convert a pointer into a member. I'm also still trying to decide if the rules for adjectives really apply here; "pointer to member type" seems more like "office chair store" (which would not be written as "office-chair store") than like, say, "office-themed store".

We have quite a few other cases (such as "pointer to function types") that follow a similar non-hyphenation pattern. These seem quite natural to me as-is.

Reply:
We decided that we can get over this. Fine as written.